src/dashboard/basicPreferences.ts

```diff
@@ -160,3 +184,20 @@ async function getDeletionUrlForServer (
   }
   return null
 }
+
+function createSection (
+  container: Element,
+  dom: HTMLDocument,
+  title: string
+): Element {
+  const section = container.appendChild(dom.createElement('div'))
+  section.style.border = '0.3em solid #418d99'
+  section.style.borderRadius = '0.5em'
+  section.style.padding = '0.7em'
+  section.style.marginTop = '0.7em'
```

Comment:
Pull the style from  `UI.style.globalNavMenuItemStyle` or something ?

Reply:
An idea. or not.

Reply:
I cannot find `globalNavMenuItemStyle` anywhere (double-checked style.js), so will discard this for now.